CGAL_ImageIO/src/CGAL_ImageIO/gis_impl.h

```diff
@@ -143,7 +143,7 @@ int writeGis( char *name, _image* im) {
 		if ( j<n && i<size ) sprintf( str+strlen(str), " " );
 	      }
 	      sprintf( str+strlen(str), "\n" );
-	      res = ImageIO_write( im, str, strlen( str )  );
+	      res = static_cast<int>(ImageIO_write( im, str, strlen( str )));
 	      if ( res  <= 0 ) {
```

Comment:
Ok. I'll have to modify `_writeInrimageData` in the process, but I guess it's alright.